monero_utils/MyMoneroCoreCpp_WASM.js

```diff
@@ -5,7 +5,7 @@ var MyMoneroCoreCpp = (function() {
 function(MyMoneroCoreCpp) {
```

Comment:
I'm not sure how this repo works, but should we incldue wasm changes in each commit? .wasm file shouldn't be commited at all I think.

Reply:
That's interesting. I think your idea could be inspired by `react-native-fast-crypto` when all built files are only included in the released package and would be ignored in the git repository.
Hi @RyanZim , how's about your idea?

Reply:
FYI @gutenye

Reply:
At this point, WASM changes in git are the normal procedure; if we want to change that, let's discuss in slack.